### PR TITLE
[ping-sender] adding missing doxygen tags

### PR DESCRIPTION
**Pull request by gavinmcgovern**

### CODE REVIEW

File: include/openthread/ping_sender.h [hunk 1]
```diff
@@ -45,6 +45,16 @@
 extern "C" {
 #endif
 
+/**
+ * @addtogroup api-ping-sender
+ *
+ * @brief
+ *   This file includes the OpenThread API for the ping sender module.
+ *
+ * @{
```
Comment (abtink) [inline include/openthread/ping_sender.h:54]: I think we need a matching comment at the end of the header file, closing `@}`:
```
/**
 * @}
 *
 */
```

**Review by abtink — COMMENTED** [review]
Looks good. Thanks 👍 
One comment about closing `@{`

**Comment by gavinmcgovern** [comment]
@googlebot I signed it!

**Comment by gavinmcgovern** [comment]
@googlebot I consent.

**Review by abtink — APPROVED** [review]
LGTM. Thanks. 👍

**Comment by gavin** [comment]
@googlebot I consent.

**Review by jwhui — APPROVED** [review]

**Pull request merged**